crates/holochain_sqlite/src/sql/cell/select_store_record.sql

```diff
@@ -0,0 +1,8 @@
+SELECT
+  hash, authored_timestamp
+FROM
+  DhtOp
+WHERE
+  validation_stage = 3
+  AND validation_status IS NOT NULL
+  AND DhtOp.type = :dht_op_type
```

Comment:
this and the `select_store_entry` sql are identical except for the name of the parameter here

Reply:
actually there seem to be 3 pairs of sql files that could be shared in this PR

Reply:
Yes, I noticed that too. I opted for duplicating every update query, so that the pairs of select/update in the code would use the same query and param names. I also didn't tidy up the updates, because the integration workflow refactor will get rid of all those special queries and just set integrated whatever is validated.

If you prefer sharing the SQL files over looking at the same names in the code, I'm happy to do that.

Reply:
Much less relevant now that the selects are gone.